### PR TITLE
Introduce Greek language

### PR DESCRIPTION
**Pull request by dkoluris**
Introduce Greek language from a native speaker living in Athens/Greece.

### CODE REVIEW

**Comment by Aldo97** [comment]
You should set the file encoding and line ending in XCode to UTC-8 and LF.
86613195-d0d01280-bfe3-11ea-93d9-8925c67172e5

**Comment by dkoluris** [comment]
Good catch, thank u

> You should set the file encoding and line ending in XCode to UTF-8 and LF.
> 86613195-d0d01280-bfe3-11ea-93d9-8925c67172e5

File: HeliPort.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -65,6 +65,7 @@
 
 /* Begin PBXFileReference section */
 		2B879D3A8D737E5EF06C350F /* Pods-HeliPort.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-HeliPort.release.xcconfig"; path = "Target Support Files/Pods-HeliPort/Pods-HeliPort.release.xcconfig"; sourceTree = "<group>"; };
+		42979EFF24B3A27200F15FBD /* el */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = el; path = el.lproj/Localizable.strings; sourceTree = "<group>"; };
```
Comment (williambj1) [inline HeliPort.xcodeproj/project.pbxproj:68]: Please move this line down to L75

File: HeliPort/Appearance/el.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+"No Status Information Avaliable" = "Δεν Υπάρχουν Πληροφορίες Κατάστασης";
+"None" = "Κανένα";
+"Open Network Preferences..." = "Άνοιγμα Προτιμήσεων Δικτύου...";
+"Open Wireless Diagnostics..." = "Άνοιγμα Ασύρματων Διαγνωστικών...";
+"Password:" = "Συνθηματικό:";
+"Password" = "Συνθηματικό";
+"Quit HeliPort" = "Έξοδος HeliPort";
+"Reachable" = "Προσβάσιμο";
+"Remember this network" = "Απομνημόνευση δικτύου";
+"Security:" = "Τύπος Ασφαλείας";
```
Comment (williambj1) [inline HeliPort/Appearance/el.lproj/Localizable.strings:27]: Missing colon?

**Comment by williambj1** [comment]
CC @dkoluris

**Pull request merged**